src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt

```diff
@@ -137,6 +142,57 @@ object SonarLintIntelliJClient : SonarLintRpcClientDelegate {
         suggestionsByConfigScopeId.forEach { (configScopeId, suggestions) -> suggestAutoBind(findProject(configScopeId), suggestions) }
     }
 
+    override fun suggestConnection(suggestionsByConfigScope: MutableMap<String, MutableList<ConnectionSuggestionDto>>?) {
+        if (suggestionsByConfigScope != null && suggestionsByConfigScope.size == 1) {
```

Comment:
`suggestionsByConfigScope != null` can be removed as the parameter should not be nullable